src/PhpDocParser/PhpParser/SmartPhpParserFactory.php

```diff
@@ -43,14 +42,12 @@ private function createPHPStanParser(Parser $parser): CachedParser
         $nameResolver = new NameResolver();
         $variadicMethodsVisitor = new VariadicMethodsVisitor();
         $variadicFunctionsVisitor = new VariadicFunctionsVisitor();
-        $propertyHookNameVisitor = new PropertyHookNameVisitor();
```

Comment:
@TomasVotruba on PHPStan 2.1.0, `PropertyHookNameVisitor` exists, but removed on PHPStan 2.1.1, this patch include it